src/queues/job-queue.js

```diff
@@ -1,7 +1,7 @@
 const {newsJob} = require('../processes/news-process');
 const {scoringProcessJob} = require('../processes/scoring-process');
 const {scoringDailyProcessJob} = require('../processes/scoring-daily-process');
-const {deleteActivityProcessJob} = require('../processes/delete-activity-process');
+// const {deleteActivityProcessJob} = require('../processes/delete-activity-process');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out import statement for `deleteActivityProcessJob` has been replaced with no new import. This could potentially lead to a reference error if `removeActivityProcessJob` is not defined elsewhere in the code.
```diff
- // const {deleteActivityProcessJob} = require('../processes/delete-activity-process');
+ const {removeActivityProcessJob} = require('../processes/remove-activity-process');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->